packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.2
+
+* Fix `PurchaseStatus` return `error` when user upgrades subscription by deferred proration mode.
```

Comment:
nit: prefer using present tense in CHANGELOG as per new [guidelines](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style).
```suggestion
* Fixes the `purchaseStream` incorrectly reporting `PurchaseStatus.error` when user upgrades subscription by deferred proration mode.
```